ios/engine/KMEI/KeymanEngine/Classes/KMManager.m

```diff
@@ -1,4099 +0,0 @@
-//
-//  KMManager.m
-//  Keyman Engine
-//
-//  Created by Patrick Weekes on 12-05-16.
-//  Updated by Serkan Kurt on 17/05/2013.
-//  Copyright (c) 2017 SIL International. All rights reserved.
-//
-
-#import "KMManager.h"
-#import "KMManager+Internal.h"
-#import "KMInputViewController.h"
-#import "KMURLProtocol.h"
-#import "KMTextField.h"
-#import "KMTextView.h"
-#import "KMPopoverView.h"
-#import "KMKeyPreviewView.h"
-#import "KMSubKeysView.h"
-#import "KMKeyboardMenuView.h"
-#import "KeyboardPickerViewController.h"
-#import "UIView+Helpers.h"
-#import <QuartzCore/QuartzCore.h>
-#import <CoreText/CoreText.h>
-#import <CommonCrypto/CommonDigest.h>
-#import "UIButton+Helpers.h"
-#import <objc/message.h>
-
-#pragma mark - Constants
-
-static BOOL enableWebKit = YES;
-
-// Keyman system-wide keyboard
-static BOOL kIsKeymanSystemKeyboard = NO;
-
-// Group Identifier
-static NSString *applicationGroupIdentifier = @"";
-
-// Notifications
-NSString *const kKeymanLanguagesUpdatedNotification = @"KeymanLanguagesUpdatedNotification";
-NSString *const kKeymanLanguagesDownloadFailedNotification = @"kKeymanLanguagesDownloadFailedNotification";
-NSString *const kKeymanKeyboardDownloadStartedNotification = @"kKeymanKeyboardDownloadStartedNotification";
-NSString *const kKeymanKeyboardDownloadCompletedNotification = @"kKeymanKeyboardDownloadCompletedNotification";
-NSString *const kKeymanKeyboardDownloadFailedNotification = @"kKeymanKeyboardDownloadFailedNotification";
-NSString *const kKeymanKeyboardChangedNotification = @"kKeymanKeyboardChangedNotification";
-NSString *const kKeymanKeyboardLoadedNotification = @"kKeymanKeyboardLoadedNotification";
-NSString *const kKeymanKeyboardPickerDismissedNotification = @"kKeymanKeyboardPickerDismissedNotification";
-NSString *const kKeymanKeyboardRemovedNotification = @"kKeymanKeyboardRemovedNotification";
-NSString *const kKeymanSubKeysMenuWillShowNotification = @"kKeymanSubKeysMenuWillShowNotification";
-NSString *const kKeymanSubKeysMenuDismissedNotification = @"kKeymanSubKeysMenuDismissedNotification";
-NSString *const kKeymanDebugLogNotification = @"kKeymanDebugLogNotification";
-
-// External dictionary keys
-NSString *const kKeymanIdKey = @"id";
-NSString *const kKeymanNameKey = @"name";
-NSString *const kKeymanKeyboardIdKey = @"kbId";
```

Comment:
The "external dictionary keys" section here as well as a few subsequent parts were moved to Constants.swift.  Just a note for anyone else seeking to review the KMManager monolith.